components/textdiff/textdiff.js

```diff
@@ -43,7 +43,7 @@ var Type = function() {
   var textDiff = 'textdiff'
 
   this.text = ko.observable("Default");
-  this.value = ko.observable(textDiff);
+  this.value = ko.observable(ungit.config.diffType||textDiff);
```

Comment:
in `.../source/config.js` can we add this variable along with standard commenting?

Also can we add quick validation where if wrong value is passed in, default to text diff with a warning log?

I think that would make this change more useable